contracts/core/Controller.sol

```diff
@@ -391,6 +412,17 @@ contract Controller is Initializable, OwnableUpgradeSafe, ReentrancyGuardUpgrade
         emit CallRestricted(callRestricted);
     }
 
+    /**
```

Comment:
what do we think about removing `setLiquidationRestriction` and making it such that setting the liquidation manager to the zero address is synonymous with saying liquidation is permissionless

Reply:
good point - addressed